embed/index.html

```diff
@@ -62,16 +62,15 @@ <h1 class="mt-5">Update</h1>
                             A filesystem update might cause a loss of all configuration data on the filesystem. Make sure you have a backup of your configuration before updating it. Backups can be done via the webinterface of Pixelix.
                         </div>
                         <div class="alert alert-warning" role="alert">
-                            A filesystem update will disable the HomeAssistant MQTT discovery feature. After the update, you have to re-enable it manually.
-                            This prevents that the welcome plugin will be discovered, after a filesystem update.
+                            Before doing a filesystem update the HomeAssistent MQTT automatic discovery feature of Pixelix needs to be disabled. To do this, switch back to Pixelix, open the Update submenu and click the 'Go to PixelixUpdater' button. After pressing the button a popup will ask you wether to disable the automatic discovery or not. Press yes to disable the automatic discovery.

```

Comment:
Is this still necessary? Because you need to confirm on Pixelix side, that HA support will be disabled for filesystem update.